pkg/storage/tsdb/config.go

```diff
@@ -60,6 +60,7 @@ func (d *DurationList) String() string {
 	return strings.Join(values, ",")
 }
 
+//nolint:staticcheck
```

Comment:
What's the error here if we remove this?

Reply:
This was my fault....looks like it got added accidentally and was not needed at all. Good catch. This has been cleaned up and I validated the other //nolint messages and they appear to all be valid.